directives/submit.py

```diff
@@ -111,6 +108,14 @@ def run(self):
         })
         self.set_assistant_permissions(data)
 
+        if data.get('title|i18n'):
+            if config_title:
+                data['title|i18n'][env.config.language] = config_title
+        else:
+            data['title'] = env.config.submit_title.format(
+                key_title=key_title, config_title=config_title
+            )
```

Comment:
Currently the `env.config.submit_title` uses always the `config_title` in the default version of a-plus-rst-tools. When the `config_title` is not defined, string 'None' is used. I think this is never the way the `title` is initiated to behave. By default rst-tools could use `key_title`, if `config_title` is not defined.